controllers/ocsinitialization/ocsinitialization_controller.go

```diff
@@ -423,3 +466,160 @@ func (r *OCSInitializationReconciler) reconcileUXBackendService(initialData *ocs
 
 	return nil
 }
+
+func (r *OCSInitializationReconciler) reconcilePrometheusKubeRBACConfigMap(initialData *ocsv1.OCSInitialization) error {
+
+	var err error
+	prometheusKubeRBACConfigMap := &corev1.ConfigMap{}
+	prometheusKubeRBACConfigMap.Name = templates.PrometheusKubeRBACPoxyConfigMapName
```

Comment:
PrometheusKubeRBACPoxyConfigMapName -> PrometheusKubeRBACProxyConfigMapName

Reply:
I think this is defined inside prometheous.go file itself no need to start the const name with Prometheous

Reply:
Have changed the typo. 
For the second comment the variable is being used in ocsinitialization controller as well. Hence keeping the prometheus prefix